project/src/pages/property-page/property-page.tsx

```diff
@@ -1,7 +1,34 @@
-import LogoComponent from '../../components/app/logo-component/logo-component';
-import CredentialComponent from '../../components/app/credential-component/credential-component';
+import LogoComponent from '../../components/logo-component/logo-component';
+import CredentialComponent from '../../components/credential-component/credential-component';
+import FeedbackFormComponent from '../../components/feedback-form-component/feedback-form-component';
+import ReviewComponent from '../../components/review-component/review-component';
+import {useParams} from 'react-router-dom';
+import {OffersType} from '../../types/offersType';
+import {ReviewsType} from '../../types/reviewsType';
 
-function PropertyPage() {
+type PropertyPageProps = {
+  offers: OffersType[];
+  reviews: ReviewsType[];
+}
+
+type ImageProps ={
+  image: string;
+}
+
+const ImageComponent = ({image}: ImageProps): JSX.Element => (
+  <div className="property__image-wrapper">
+    <img className="property__image" src={image} alt=""/>
+  </div>);
+
+const renderIsPro = (offer: OffersType | undefined): JSX.Element | undefined => {
+  if (offer?.host.isPro) {
+    return <span className="property__user-status">Pro</span>;
+  }
+};
+
+function PropertyPage({offers, reviews}: PropertyPageProps) {
+  const id = useParams().id;
+  const offer = offers.find((o: OffersType) => o.id === Number(id));
   return (
```

Comment:
не нужно так сокращать имя праметра, во первых есть критерий (Б6), а во вторых это сильно читаемость ухудшает, и тут кстати тип можно не писать
```suggestion
  const offer = offers.find((offer) => offer.id === Number(id));
  return (
```

Reply:
плюс не плохо бы добавить проверку , чтобы если offer в итоге не нашёлся, редиректило на 404